config/default_jobs.yaml

```diff
@@ -189,6 +189,80 @@ weekend_jobs:
     cur: null
     predicates_path: null
 
+# Jobs that run on the weekend targeting the test environment. Note
+# that the subset of tables we have access to in test is more limited
+# than in prod i.e. we only have access to the tables below
+weekend_jobs_test:
+  addn:
+    table_name: ias.addn
```

Comment:
Note that the schema is also slightly different here, so glad we made `table_name` a job option.